### ISSUE TITLE
Luet hangs under certain repository configuration setup

### ISSUE THREAD

**Issue by davidcassany** [rancher-sandbox/cOS-toolkit#295]
**Describe the bug**

The issue raised in `raw-img` job in GHA and I easily managed to reproduce it locally. The image builder scripts hangs on luet install if luet includes a local repository on top of the system ones (`/etc/luet/repos.conf.d`). In image builder script a `.luet.yaml` file is appended into the working directory to include a local disk repository, then if luet configuration includes other repositories from `/etc/luet/repos.conf.d` it hangs on `luet install`.

**To Reproduce**

An easy way to reproduce it is to mimic locally all the steps done in `raw-img` job inside a `opensuse/leap:15.3` container without removing (or disabling) the `/etc/luet/repos.conf.d/` contents. There one can see how `luet install` hangs. Having the system repositories from `/etc/luet/repos.conf.d` disabled is enough as a workaround.

**Expected behavior**

Luet install works without issues

**Comment by Itxaka** [comment]
I have also experienced this when building the raw_img locally, I had to backup my repos and left luet without any for the raw_image in order to be able to build

My local repos:

```
# itxaka @ zeus in ~/projects/cOS-toolkit on git:share_image_azure o [11:31:00] 
$ cat /etc/luet/repos.conf.d/bak/luet.yml 
name: "luet"
description: "Luet official Repository"

type: "http"
enable: true
cached: true
priority: 1
urls:
- "https://raw.githubusercontent.com/Luet-lab/luet-repo/gh-pages"
# itxaka @ zeus in ~/projects/cOS-toolkit on git:share_image_azure o [11:43:41] 
$ cat /etc/luet/repos.conf.d/bak/mocaccino-extra-stable.yml 
name: "mocaccino-extra-stable"
description: "MocaccinoOS extra Repository (stable)"
type: "http"
enable: true
cached: true
priority: 2
urls:
  - "https://get.mocaccino.org/mocaccino-extra-stable"

# itxaka @ zeus in ~/projects/cOS-toolkit on git:share_image_azure o [11:43:53] 
$ cat /etc/luet/repos.conf.d/bak/mocaccino-repository-index.yml 
name: "mocaccino-repository-index"
description: "MocaccinoOS Repository Index"
type: "http"
enable: true
cached: true
priority: 1
urls:
- "https://raw.githubusercontent.com/mocaccinoOS/repository-index/gh-pages"
- "https://get.mocaccino.org/mocaccino-repository-index"


```

**Comment by mudler** [comment]
I think I know what's going on: `--system-target` needs an absolute path. I've traced it back to [docker.Untar](https://github.com/mudler/luet/blob/master/vendor/github.com/docker/docker/pkg/archive/archive.go#L942):  looks like the issue is propagated down there as it is passed by in relative form, and there it loops.  I'll add a check on luet side of things to transform it in absolute if not explicitly passed

**Comment by mudler** [comment]
A fix for it is there: https://github.com/mudler/luet/commit/9d1594c0365142400d6057ffc43cb71eefd321ad I'm going to tag a release with also other bugfixes

**Comment by mudler** [comment]
should be fixed in 0.17.3

**Comment by Itxaka** [comment]
can confirm that its fixed, nice!
